process/headerCheck/headerSignatureVerify.go

```diff
@@ -20,6 +21,8 @@ import (
 	"github.com/multiversx/mx-chain-go/sharding/nodesCoordinator"
 )
 
+const headerWaitDelayAtTransition = 5 * time.Millisecond
```

Comment:
I think we can add a longer delay, thinking about 50ms, otherwise it can be too intensive on the storage.